README.md

```diff
@@ -25,6 +38,27 @@ can be added to Cloudflare using `wrangler` like so:
 wrangler secret put DB
 ```
 
+The value to enter is the connection string from the pgEdge console. This can be found on the Integrations
+tab of your database under the Cloudflare Workers integration, or on the database details page under the 
+"Connect to your database" section.  The "Nearest Node" tab will provide you with the connection string.
+
+### Add your local secret
+
 Additionally, you should make a `.dev.vars` file and set the `DB` environment variable there as well
 so that you can run the worker locally.
 
+```bash
+DB=postgres://user:pass@host/dbname?sslmode=required
+```
+
+### Run your worker locally
+
+You can run your worker locally using `wrangler`:
+
+```bash
+wrangler dev
```

Comment:
If wrangler were in the dev dependencies, this could be a script in `package.json`.

Reply:
I could do that, though Cloudflare docs are pretty opinionated (obviously) about just using wrangler as its own CLI without wrapping it in tooling.

Reply:
Again, looking at the repos above, they also tend to not have wrangler commands wrapped in npm scripts.